zambia_locations.csv

```diff
@@ -0,0 +1,174 @@
+Name,Type,Parent
```

Comment:
Do we intend to commit `zambia_locations.csv` to source control? Up til now it has been decoupled.

Reply:
No, will remove